web/static/js/reducers/stage.js

```diff
@@ -1,5 +1,7 @@
 const stage = (state = "idea-generation", action) => {
   switch (action.type) {
+    case "SET_INITIAL_STATE":
+      return action.initialState.stage
```

Comment:
a) Empty string definitely works! b) and yeah, that's definitely an approach worth exploring this week 👍